graphql/graphql.go

```diff
@@ -151,6 +151,20 @@ func (l *Log) Data(ctx context.Context) hexutil.Bytes {
 	return l.log.Data
 }
 
+// Access Tuple represents EIP-2930
+type AccessTuple struct {
+	address     common.Address
+	storageKeys *[]common.Hash
+}
```

Comment:
We already have 
```
// AccessList is an EIP-2930 access list.
type AccessList []AccessTuple

// AccessTuple is the element type of an access list.
type AccessTuple struct {
	Address     common.Address `json:"address"        gencodec:"required"`
	StorageKeys []common.Hash  `json:"storageKeys"    gencodec:"required"`
}
```
In `core/types`. Is there any reason not to just use those?

Reply:
Ill add testcases to graphql_test.go for type and accessList properties retrieval.  My initial line of thinking was separating the graphql and core object models, but I directly use the tuples if that is preferred.